src/components/LocationPageHeader/LocationPageHeader.js

```diff
@@ -86,48 +81,73 @@ const LocationPageHeader = ({ projections }) => {
   return (
-    <StyledLocationPageHeaderWrapper condensed={isEmbed}>
-      <StyledLocationPageHeaderInner condensed={isEmbed}>
-        <StyledStateImageWrapper>
-          <StateCircleSvg
-            actionBackgroundFill={COLORS.LIGHTGRAY}
-            state={projections.stateCode}
-            fillColor={fillColor}
-            hasAction={true}
-          />
-        </StyledStateImageWrapper>
-        <StyledStateCopyWrapper>
-          <div>
-            <HeaderTitle>
-              <HeaderHighlight color={textColor}>
-                <LocationPageHeading projections={projections} />
-              </HeaderHighlight>
-            </HeaderTitle>
-            {!isEmbed ? <LocationSummary projections={projections} /> : ''}
-            {projections.isCounty && !isEmbed && (
-              <HeaderSubCopy>
-                <strong>County data is currently in beta. </strong>
-                <span>
-                  Because counties don’t report hospitalizations, our forecasts
-                  may not be as accurate. See something wrong?{' '}
-                </span>
-                <a
-                  href="https://forms.gle/NPsLcFnrvfS1kqkn9"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                >
-                  Please let us know.
-                </a>
-              </HeaderSubCopy>
-            )}
-          </div>
-        </StyledStateCopyWrapper>
-      </StyledLocationPageHeaderInner>
-    </StyledLocationPageHeaderWrapper>
+    <>
+      <StyledLocationPageHeaderWrapper bgColor={fill} condensed={isEmbed}>
+        <StyledLocationPageHeaderInner condensed={isEmbed}>
+          <StyledStateCopyWrapper>
+            <div>
+              <HeaderTitle>
+                <HeaderHighlight>
+                  <LocationPageHeading projections={projections} />
+                </HeaderHighlight>
+              </HeaderTitle>
+              <StyledStatusBadge color={fill}>
+                Status goes here
+              </StyledStatusBadge>
+              {!isEmbed ? <LocationSummary projections={projections} /> : ''}
+              {projections.isCounty && !isEmbed && (
+                <HeaderSubCopy>
+                  <strong>County data is currently in beta. </strong>
+                  <span>
+                    Because counties don’t report hospitalizations, our
+                    forecasts may not be as accurate. See something wrong?{' '}
+                  </span>
+                  <a
+                    href="https://forms.gle/NPsLcFnrvfS1kqkn9"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    Please let us know.
+                  </a>
+                </HeaderSubCopy>
+              )}
+            </div>
+          </StyledStateCopyWrapper>
+        </StyledLocationPageHeaderInner>
+      </StyledLocationPageHeaderWrapper>
+      <StyledDashboardWrapper>
```

Comment:
I def prefer these to be their own components so that the files are easy to understand? https://github.com/covid-projections/covid-projections/pull/599